apps/volontulo/tests/models/test_userprofile.py

```diff
@@ -18,37 +17,38 @@ def setUp(self):
         u"""Set up each test."""
 
         # volunteer user
-        self.volunteer_user = UserProfile.objects.create(
-            user=User.objects.create_user(
-                username='volunteer@example.com',
-                email='volunteer@example.com',
-                password='volunteer',
-            ),
-            is_administrator=False,
+        user = User.objects.create_user(
```

Comment:
@riv2q DDF support had been merged, but I believe we could skip using it in this PR, as it is already raised.